app/src/main/java/org/ole/planet/myplanet/ui/dashboard/DashboardActivity.java

```diff
@@ -106,6 +109,9 @@ protected void onCreate(Bundle savedInstanceState) {
         navigationView.setVisibility(new UserProfileDbHandler(this).getUserModel().getShowTopbar() ? View.VISIBLE : View.GONE);
         headerResult = getAccountHeader();
         createDrawer();
+        if(getWindowManager().getDefaultDisplay().getRotation() == Surface.ROTATION_0 || getWindowManager().getDefaultDisplay().getRotation() == Surface.ROTATION_180) {
```

Comment:
@anastasia21112  ``` if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_PORTRAIT){ }``` use this one , instead of checking rotation.